### PR TITLE
ColumnSQL automatically set IsLazyLoading

### PR DESCRIPTION
**Pull request by riadmen**

### CODE REVIEW

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java [hunk 1]
```diff
@@ -335,6 +335,13 @@ private static void updateNameOrDocumentNoColumn(final I_AD_Column column)
 		column.setFieldLength(40);
 	}
 
+	@CalloutMethod(columnNames = { I_AD_Column.COLUMNNAME_ColumnSQL })
+	public static void upadteIsLazyLoading(final I_AD_Column column)
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java:339]:
```suggestion
	public static void updateIsLazyLoading(final I_AD_Column column)
```

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java [hunk 2]
```diff
@@ -335,6 +335,13 @@ private static void updateNameOrDocumentNoColumn(final I_AD_Column column)
...
+	{
+		IADTableDAO adTableDAO = Services.get(IADTableDAO.class);
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java:341]:
```suggestion
		final IADTableDAO adTableDAO = Services.get(IADTableDAO.class);
```

**Review by metas-ts — APPROVED** [review]

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java [hunk 3]
```diff
@@ -335,6 +335,13 @@ private static void updateNameOrDocumentNoColumn(final I_AD_Column column)
 		column.setFieldLength(40);
 	}
 
+	@CalloutMethod(columnNames = { I_AD_Column.COLUMNNAME_ColumnSQL })
+	public static void updateIsLazyLoading(final I_AD_Column column)
```
Comment (teosarca) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java:339]: not sure if is OK to have it `static`.....

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java [hunk 4]
```diff
@@ -335,6 +335,13 @@ private static void updateNameOrDocumentNoColumn(final I_AD_Column column)
...
+	{
+		final IADTableDAO adTableDAO = Services.get(IADTableDAO.class);
```
Comment (teosarca) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java:341]: pls declare it as a field

Comment (metas-ts) [reply backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/column/callout/AD_Column.java]: done

**Review by metas-ts — APPROVED** [review]

**Pull request merged**